todo/middlewares/jwt_auth.py

```
@@ -73,25 +82,61 @@
+            user_data = {
+                "user_id": payload["user_id"],
+                "google_id": payload["google_id"],
+                "email": payload["email"],
+                "name": payload.get("name", ""),
+            }
+
+            new_access_token = generate_google_access_token(user_data)
+
+            self._set_google_user_data(request, payload)
+
+            request._new_access_token = new_access_token
```

Pylint (3.3.7)

[warning] Access to a protected member _new_access_token of a client class

(W0212)